src/App.tsx

```diff
@@ -1,14 +1,63 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import { getTodos, getUser } from './api';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+import { ByStatus } from './types/ByStatus';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [filtredByStatus, setFiltredByStatus] = useState<ByStatus>(
+    ByStatus.all,
+  );
+  const [query, setQuery] = useState<string>('');
+  const [showModal, setShowModal] = useState<boolean>(false);
+
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+  const [isLoadingUser, setIsLoadingUser] = useState(false);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+
+  useEffect(() => {
+    getTodos().then((fetchedTodos) => {
+      setTodos(fetchedTodos);
```

Comment:
it would be great here to add some `catch ` and  `finally ` blocks to handle error

Reply:
Good point